CMakeLists.txt

```diff
@@ -126,6 +126,7 @@ option(QUIC_CODE_CHECK "Run static code checkers" OFF)
 option(QUIC_OPTIMIZE_LOCAL "Optimize code for local machine architecture" OFF)
 option(QUIC_CI "CI Specific build optimizations" OFF)
 option(QUIC_RANDOM_ALLOC_FAIL "Randomly fails allocation calls" OFF)
+option(QUIC_WIRESHARK_SUPPORT "Enable Wireshark to decode packets" OFF)
```

Comment:
I think I'd prefer to just use SSLKEYLOGFILE instead of WIRESHARK. Other tools can/may use the file as well.